modules/effects/src/concat_latest_from.ts

```diff
@@ -1,5 +1,4 @@
-import { Observable, ObservedValueOf, of, OperatorFunction, pipe } from 'rxjs';
-import { concatMap, withLatestFrom } from 'rxjs/operators';
```

Comment:
We're still using `rxjs/operators` for backwards compatibility with RxJS 6.5.x, so we still need those imported from here also. And the `map` operator needs to be imported for the changes below.

Reply:
Ew. But okay.

And sorry about the mistakes. I'm just using the GitHub IDE.

Reply:
No problem. Yea, we'd like to move off support for 6.5.x also but the existing apps themselves need to be upgraded. New apps are on RxJS 7.x